### PR TITLE
Discontinue support for chandra_cmd_states

### PR DESCRIPTION
**Pull request by taldcroft**
## Description

This addresses one action in https://github.com/sot/kadi/issues/230.

## Interface impacts

Importing package raises an exception unless `SKA_ALLOW_DISCONTINUED_PACKAGES=1`.

### Unit tests

No testing required for discontinued package.

#### Importing package fails without env var
```
(ska3) ➜  cmd_states git:(master) ✗ ipython
Python 3.10.8 | packaged by conda-forge | (main, Nov 22 2022, 08:27:35) [Clang 14.0.6 ]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.8.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import chandra_cmd_states
/Users/aldcroft/miniconda3/envs/ska3/lib/python3.10/site-packages/Ska/ParseCM/ParseCM.py:8: FutureWarning: ska_parsecm is deprecated, use parse_cm instead
  warnings.warn('ska_parsecm is deprecated, use parse_cm instead', FutureWarning)
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[1], line 1
----> 1 import chandra_cmd_states

File ~/git/cmd_states/chandra_cmd_states/__init__.py:23
     21     warnings.warn(message_warn, FutureWarning)
     22 else:
---> 23     raise RuntimeError(message_warn + message_error)
     25 __version__ = ska_helpers.get_version("chandra_cmd_states")
     28 def test(*args, **kwargs):

RuntimeError: 
The chandra_cmd_states package is no longer supported and is replaced
by kadi.commands.states. Please update your code accordingly.

To temporarily continue using this package set the environment variable
SKA_ALLOW_DISCONTINUED_PACKAGES=1

In [2]: import chandra_cmd_states
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[2], line 1
----> 1 import chandra_cmd_states

File ~/git/cmd_states/chandra_cmd_states/__init__.py:23
     21     warnings.warn(message_warn, FutureWarning)
     22 else:
---> 23     raise RuntimeError(message_warn + message_error)
     25 __version__ = ska_helpers.get_version("chandra_cmd_states")
     28 def test(*args, **kwargs):

RuntimeError: 
The chandra_cmd_states package is no longer supported and is replaced
by kadi.commands.states. Please update your code accordingly.

To temporarily continue using this package set the environment variable
SKA_ALLOW_DISCONTINUED_PACKAGES=1
```
#### Setting env var allows import with a FutureWarning
```
In [3]: import os

In [4]: os.environ["SKA_ALLOW_DISCONTINUED_PACKAGES"] = "1"

In [5]: import chandra_cmd_states
/Users/aldcroft/git/cmd_states/chandra_cmd_states/__init__.py:21: FutureWarning: 
The chandra_cmd_states package is no longer supported and is replaced
by kadi.commands.states. Please update your code accordingly.

  warnings.warn(message_warn, FutureWarning)

In [6]: import chandra_cmd_states

In [7]:                                                                                                                                                      
```

### CODE REVIEW

**Review by javierggt — APPROVED** [review]
The docs workflow will fail, so we will need to make it and commit it by hand.

File: chandra_cmd_states/__init__.py [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+
+message_warn = (
+    "\n"
+    "The chandra_cmd_states package is no longer supported and is replaced\n"
+    "by kadi.commands.states. Please update your code accordingly.\n"
+)
+message_error = (
+    "\n"
+    "To temporarily continue using this package set the environment variable\n"
+    "SKA_ALLOW_DISCONTINUED_PACKAGES=1"
```
Comment (jeanconn) [inline chandra_cmd_states/__init__.py:13]: Picking nits, I think that's SKA_ALLOW_DISCONTINUED_PACKAGES='1'.  but I'm good!

**Review by jeanconn — APPROVED** [review]

**Comment by jeanconn** [comment]
Oops, I approved the wrong one.  But I'll re-approve this one soon.

**Comment by jeanconn** [comment]
For this one, looking over code that might still use Chandra.cmd_states - what is the status of pyger and spaceweather repos?

**Review by jeanconn — CHANGES_REQUESTED** [review]
Temporarily changing back my review status

**Comment by taldcroft** [comment]
Both pyger and spaceweather are not used in any production workflows and can be fixed as needed. I think pyger is no longer maintained or used. Spaceweather is my repo and I'll swap in kadi states as needed.

**Review by jeanconn — APPROVED** [review]

**Pull request merged**